debian/Dockerfile

```diff
@@ -0,0 +1,59 @@
+ARG VERSION="5.1"
+
+# Assume this is run from capstone/debian directory
+# Run in the root of the repo
+# docker build -f ./debian/Dockerfile -t packager .
+FROM debian:buster-slim
+
+# Install necessary tools for packaging
+RUN apt-get -qq update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -qq install -y \
+        fakeroot dpkg-dev dos2unix
+
+# Then install capstone from source
+RUN mkdir /capstone
+
+# Copy your project files into the container
+COPY . /capstone
+
+# Ensure make.sh is executable
+RUN dos2unix /capstone/make.sh
+RUN chmod +x /capstone/make.sh
+
+# Run the make.sh script and install
+WORKDIR /capstone/ 
+RUN ./make.sh
```

Comment:
Please use the cmake build. make is considered deprecated. Check out `BUILDING.md`

Reply:
Damn, seems like PANDA team was building it wrong the whole time, well I'll fix this in a second

https://github.com/panda-re/panda/blob/dev/Dockerfile#L38-L42

Reply:
All good. Makefiles still work. But we don't really take care of them anymore.